ingenialink/canopen/network.py

```diff
@@ -239,18 +244,21 @@ def __init__(
         if self.__device == CAN_DEVICE.PCAN.value:
             self.__connection_args["auto_reset"] = True
 
-    def scan_slaves(self) -> List[int]:
+    def scan_slaves(self) -> list[int]:
         """Scans for nodes in the network.
 
         Returns:
             Containing all the detected node IDs.
 
         """
         if (self.__device, self.__channel) not in self.get_available_devices():
-            raise ILError(
+            msg = (
                 f"The {self.__device.upper()} transceiver is not detected. "
                 "Make sure that it's connected and its drivers are installed."
             )
+            raise ILError(
```

Comment:
I don't understand this change. Why create a message variable and then raise it with a trailing comma?

Reply:
Check [EM102](https://docs.astral.sh/ruff/rules/f-string-in-exception/)
I think that the trailing comma is the formatter's work.